gslab_scons/tests/test_build_python.py

```diff
@@ -3,40 +3,124 @@
 import sys
 import os
 import shutil
+import mock
+import subprocess
 
 sys.path.append('../..')
-from gslab_scons import build_python
+import gslab_scons.builders.build_python as gs
 from gslab_scons._exception_classes import BadExtensionError
+from gslab_make.tests import nostderrout
 
-
-class testbuild_python(unittest.TestCase):
+class TestBuildPython(unittest.TestCase):
```

Comment:
I didn't see much to be gained in terms of speed or portability from changing the existing structure but I'm happy to change this if you think otherwise.